packages/cli/src/index.ts

```diff
@@ -61,24 +49,7 @@ export async function main(medplumClient: MedplumClient, argv: string[]): Promis
 
 export async function run(): Promise<void> {
   dotenv.config();
-  const baseUrl = process.env['MEDPLUM_BASE_URL'] || 'https://api.medplum.com/';
-  const fhirUrlPath = process.env['MEDPLUM_FHIR_URL_PATH'] || '';
-  const accessToken = process.env['MEDPLUM_CLIENT_ACCESS_TOKEN'] || '';
-  const tokenUrl = process.env['MEDPLUM_TOKEN_URL'] || '';
-
-  const medplumClient = new MedplumClient({
-    fetch,
-    baseUrl,
-    tokenUrl,
-    fhirUrlPath,
-    storage: new FileSystemStorage(),
-    onUnauthenticated: onUnauthenticated,
-  });
-
-  if (accessToken) {
-    medplumClient.setAccessToken(accessToken);
-  }
-  await main(medplumClient, process.argv);
```

Comment:
This logic has moved to https://github.com/medplum/medplum/pull/2197/files#diff-80e8d5a02c540100cc38f9c1240d73742010ee4b2f195e615943ac89d67d5bc7